Source/Core/Core/PowerPC/JitArm64/JitArm64_SystemRegisters.cpp

```diff
@@ -653,15 +657,17 @@ void JitArm64::crXXX(UGeckoInstruction inst)
       break;
 
     case 129:  // crandc: A && ~B
-      BIC(XA, XA, XB);
+      BIC(WA, WA, WB);
+      bits_1_to_31_are_set = true;
       break;
```

Comment:
A is either 0 or 1, right? If so `A && ~B` should result in 0 or 1 too.

Reply:
Good point!